src/widgets/HistogramWidget.js

```diff
@@ -58,6 +59,8 @@ function HistogramWidget(props) {
         filters,
         credentials,
         viewport,
+        viewportFeatures,
```

Comment:
viewportFeatures should be set to viewportFeatures[source], because the models shouldn't have logic around the sources